test/index.js

```diff
@@ -650,7 +651,7 @@ describe('mdast.process(value, options, done)', function () {
     it('should run plugins', function () {
         assert(
             mdast.use(mentions).process('@mention') ===
-            '[@mention](https://github.com/blog/821)\n'
+            '[@mention](https://github\\.com/blog/821)\n'
```

Comment:
@wooorm Your link includes two backslashes instead of one (two backslashes are needed to put it into a JS string, but the text includes only one). Commonmark actually [handles](http://spec.commonmark.org/dingus/?text=[%40mention]%28https%3A%2F%2Fgithub\\.com%2Fblog%2F821%29) escapes in links. It does not interpret escapes [in autolinks](http://spec.commonmark.org/dingus/?text=%3Chttps%3A%2F%2Fgithub\\.com%2Fblog%2F821%3E) but I think I handled that, and there're some tests for that.